public/scripts/extensions/regex/style.css

```
@@ -1,3 +1,7 @@
+[hidden] {
```

This style is not specific enough.
```suggestion
.regex_settings [hidden] {
```